RepoDb.PostgreSql/RepoDb.PostgreSql/StatementBuilders/PostgreSqlStatementBuilder.cs

```diff
@@ -505,11 +501,9 @@ public PostgreSqlStatementBuilder()
             }
 
             // Set the return value
-            var result = identityField != null ?
-                string.IsNullOrEmpty(databaseType) ?
-                    identityField.Name.AsQuoted(DbSetting) :
-                        string.Concat($"CAST({identityField.Name.AsQuoted(DbSetting)} AS {databaseType})") :
-                            primaryField != null ? primaryField.Name.AsParameter(DbSetting) : "NULL";
+            var result = identityField == null ? primaryField.Name.AsParameter(DbSetting) :
```

Comment:
Same here

Reply:
want me to do this one as well? or does it make sense with the above explanations?

Reply:
Yes, please, I think, it is necessary.

Reply:
u merged without me addressing this comment. do we need to revisit this specific change?